cves/CVE-2013-1665.yml

```diff
@@ -49,7 +49,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
Great explanation of why deserialization is not secure enough

Reply:
Thank you!